codacy/values-microk8s.yaml

```diff
@@ -0,0 +1,184 @@
+global:
+  imagePullSecrets:
+    - name: docker-credentials
+  codacy:
+    url: "<--- codacy protocol --->://<--- codacy url --->"
+    backendUrl: "<--- codacy protocol --->://<--- codacy url --->"
+  akka:
+    sessionSecret: "<--- akka session secret --->"
+  play:
+    cryptoSecret: "<--- play session secret --->"
+  filestore:
+    contentsSecret: "<--- filestore contents secret --->"
+    uuidSecret: "<--- filestore uuid secret --->"
+  cacheSecret: "<--- cache secret --->"
+
+codacy-api:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 2000Mi
+    requests:
+      cpu: 100m
+      memory: 1000Mi
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: "nginx"
+    hosts:
+    - host: '<--- codacy url --->'
+      paths:
+        - /
+
+portal:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1Gi
+    requests:
+      cpu: 100m
+      memory: 500Mi
+
+activities:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1Gi
+    requests:
+      cpu: 100m
+      memory: 500Mi
+  activitiesdb:
+    persistence:
+      enabled: true
+      size: 50Gi
+
+remote-provider-service:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 750Mi
+    requests:
+      cpu: 100m
+      memory: 300Mi
+
+hotspots-api:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1Gi
+    requests:
+      cpu: 100m
+      memory: 500Mi
+  hotspotsdb:
+    persistence:
+      enabled: true
+      size: 50Gi
+
+hotspots-worker:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1Gi
+    requests:
+      cpu: 100m
+      memory: 500Mi
+
+listener:
+  replicaCount: 1
+  resources:
+    limits:
+      cpu: 2
+      memory: 8Gi
+    requests:
+      cpu: 1
+      memory: 6Gi
```

Comment:
those values are not in-line with latest agreement, correct?

Reply:
True, they are not. I am going to update the file.